tests/test_channels.py

```diff
@@ -193,7 +193,7 @@ def test_eom_channel():
         TypeError,
         match="When defined, 'eom_config' must be a valid 'RydbergEOM'",
     ):
-        Rydberg.Global(None, None, eom_config=BaseEOM(50))
+        Rydberg.Global(None, None, mod_bandwidth=3, eom_config=BaseEOM(50))
```

Comment:
nit: Perhaps you can invert the place of this error with the one below (first condition is to have mod_bandwidth defined, second is to have eom_config defined) ?

Reply:
Hum no I am okay with the current implementation, it was just about the order in the test file:
```
    with pytest.raises(
        ValueError,
        match="'eom_config' can't be defined in a Channel without a"
        " modulation bandwidth",
    ):
        Rydberg.Global(None, None, eom_config=_eom_config)

    with pytest.raises(
        TypeError,
        match="When defined, 'eom_config' must be a valid 'RydbergEOM'",
    ):
        Rydberg.Global(None, None, mod_bandwidth=3, eom_config=BaseEOM(50))
```

Reply:
Got it, done